css/properties/scrollbar-color.json

```diff
@@ -18,7 +18,7 @@
               "version_added": false
             },
             "firefox": {
-              "version_added": "62",
+              "version_added": "63",
```

Comment:
https://bugzilla.mozilla.org/show_bug.cgi?id=1492012 says this is in Firefox 64. So I think this should be 64, or do you have other information?